src/site/content/en/blog/performance-measure-memory/index.md

```diff
@@ -0,0 +1,356 @@
+---
```

Comment:
@ulan please rename this directory to `monitor-total-memory-usage` or `performance-measurememory`

The first option is more aligned with our policy that the page URL should match the page title, the second option is more aligned with the actual name of the API (having a dash between `measure` and `memory` doesn't make sense to me)

Reply:
Renamed to `monitor-total-page-memory-usage`